app/helpers/barnardos/action_view/form_helpers.rb

```diff
@@ -90,6 +91,12 @@ def radio_group_vertical(name, legend, selection_list, value = nil, error: nil,
                 end
               )
             end
+            concat(
+              content_tag(:div,
+                          "#{name.to_s.humanize} #{error[0]}",
+                          class: 'radio-group__error'
+              )
+            ) if error.present?
```

Comment:
If testing for `nil`, can just say `if error` here (any truthy value will do) – `present` is for when you need to test for nil or empty, in the case of a `String` that might be `""` or an array that might be `[]`